.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
We could also use a per-language version of this, but give we shouldn't expect much more in this repo than .rs, .toml, some .json, and a handful of .yml files for pipelines this is probably fine for now.